charts/pega/templates/_pega_hpa.tpl

```diff
@@ -22,14 +22,17 @@ spec:
   maxReplicas: 5
   {{- end }} 
   metrics:
+  {{- if (hasKey .hpa "enableCpuTarget" | ternary .hpa.enableCpuTarget true) }}
   - type: Resource
     resource:
       name: cpu
       {{- if .hpa.targetAverageCPUUtilization }}
       targetAverageUtilization: {{ .hpa.targetAverageCPUUtilization }}
       {{- else }}
       targetAverageUtilization: 70
-      {{- end }}  
+      {{- end }}
+  {{- end }}
+  {{- if (hasKey .hpa "enableMemoryTarget" | ternary .hpa.enableMemoryTarget true) }}
```

Comment:
@taz-pega-work Looks like these changes are already in [master ](https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/templates/_pega_hpa.tpl) I'm not sure why is the diff still showing it as a new addition. Can you please confirm on how did you merge this branch with upstream master branch?